src/main/java/org/spongepowered/common/data/type/SpongeBodyPart.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.common.data.type;
+
+import org.spongepowered.api.data.type.BodyPart;
+import org.spongepowered.api.text.translation.Translation;
+import org.spongepowered.common.SpongeCatalogType;
+
+public class SpongeBodyPart extends SpongeCatalogType implements BodyPart {
+
+    public SpongeBodyPart(String id) {
+        super(id);
+    }
+
+    @Override
+    public Translation getTranslation() {
+        return null; //unsure what translation to use and SpongeCatalogType.Translatable requires a non-null translation
```

Comment:
Yeah I noticed that there is no translation for this, it's now been removed in the `bleeding` branch in SpongeAPI (https://github.com/SpongePowered/SpongeAPI/commit/87585f18a87d9c748d1a2ed4de3ac25af75895b9) so this could probably just throw `UnsupportedOperationException` for now